.github/workflows/python-package.yml

```diff
@@ -49,6 +49,10 @@ jobs:
       run: |
         source ./venv/bin/activate
         make lint
+    - name: Test
+      run: |
+        source ./venv/bin/activate
+        pytest -svx raiden_contracts/tests/test_token_network_registry.py
```

Comment:
Why do we need a separate step for this test?

Reply:
Ah no, this can be removed now that it's working.